EnglishFilter/sections/specific.txt

```diff
@@ -264,6 +264,9 @@ lookmovie.io##.bottom-message-container
 lookmovie.io##div[class$="-page-ads-wrapper"]
 lookmovie.io##.notifyjs-corner
 pixiv.net##a[target="premium_noads"]
+||babesinporn.com/flr.js
+babesinporn.com##.list > div.item > a[target="_parent"] > img[width="240"]
+babesinporn.com##body > div[style^="display: block; position: fixed; z-index:"]
```

Comment:
```
||content*.babesinporn.com/images/*/istripper/
||content*.babesinporn.com/images/*/strp_v*.js
```
You can block requests, also it's better for second rule. You have also missed the `playboy` ad. `||content*.babesinporn.com/images/*/banners/`

Reply:
@zloyden that istripper and banners are shared with many similar sites. I've added a common rule to uAssets recently, will open PR.

Reply:
I have noticed it but we don't make these rules `generic` right away.
For now there are common rules with domains.

Reply:
I agree not to make generic. As you see in my PR, just adding domains. Apparently flr.js is not common, but I'll add few more domains to banners. OTOH I'm not sure about /bload, it seems usually not loaded. You can find inline script to load `/bloader` conditionally in e.g. `https://www.rossoporn.com/grace-beautiful-nude-girl`: